src/PrimeTower.cpp

```diff
@@ -40,9 +47,32 @@ PrimeTower::PrimeTower()
     const Scene* scene_pointer = &scene; //Communicate to lambda via pointer to prevent copy.
     std::stable_sort(extruder_order.begin(), extruder_order.end(), [scene_pointer](const unsigned int& extruder_nr_a, const unsigned int& extruder_nr_b) -> bool
     {
+        bool retval;
+
         const Ratio adhesion_a = scene_pointer->extruders[extruder_nr_a].settings.get<Ratio>("material_adhesion_tendency");
         const Ratio adhesion_b = scene_pointer->extruders[extruder_nr_b].settings.get<Ratio>("material_adhesion_tendency");
-        return adhesion_a < adhesion_b;
+
+        EPlatformAdhesion adhesion_type = scene_pointer->current_mesh_group->settings.get<EPlatformAdhesion>("adhesion_type");
+        const unsigned int adhesion_extr = scene_pointer->current_mesh_group->settings.get<ExtruderTrain&>("adhesion_extruder_nr").extruder_nr;
+
+        if (adhesion_type == EPlatformAdhesion::RAFT)
+        {
+            retval = (extruder_nr_a == adhesion_extr);
```

Comment:
Instead of this `retval`, it's more semantic to just use `return`.
```suggestion
            return extruder_nr_a == adhesion_extr;
```

Reply:
Well ... this is a very, very long story ! E.g. https://softwareengineering.stackexchange.com/questions/118703/where-did-the-notion-of-one-return-only-come-from.
I started coding (when I was 10 or so) in the era of 'structured programming' and in my code I'm still following those principles (unless I'm using C++'s exceptions, but doesn't seem to be the case in CuraEngine).
Anyway, I'm not fighting for a line of code out of thousands, so ... ok :-)